fv/wireguard_test.go

```diff
@@ -78,6 +84,19 @@ var _ = infrastructure.DatastoreDescribe("_BPF-SAFE_ WireGuard-Supported", []api
 			Skip("Skipping Wireguard supported tests.")
 		}
 
+		// Enable Wireguard module debugging.
+		utils.Run("sudo", "sh", "-c", "echo module wireguard +p > /sys/kernel/debug/dynamic_debug/control")
+
+		// Start a process tailing the dmesg log.
+		ctx, cancel := context.WithCancel(context.Background())
+		dmesgCmd = exec.CommandContext(ctx, "sudo", "dmesg", "-wH")
```

Comment:
If you want slightly more readable timestamps you can use `dmesg -w --time-format iso`.

Reply:
If you don't mind I'll leave that to a later iteration, to save a CI cycle and because the output already looks like it would be useful in case of the flake recurring:
```
[ +27.394523] wireguard: wireguard.cali: Sending handshake initiation to peer 443 (172.17.0.5:51820)
[  +0.000073] wireguard: wireguard.cali: Sending handshake initiation to peer 444 (172.17.0.4:51820)
[  +0.000121] wireguard: wireguard.cali: Receiving handshake initiation from peer 444 (172.17.0.4:51820)
[  +0.000001] wireguard: wireguard.cali: Sending handshake response to peer 444 (172.17.0.4:51820)
[  +0.000088] wireguard: wireguard.cali: Keypair 135 created for peer 444
[  +0.000071] wireguard: wireguard.cali: Receiving handshake initiation from peer 443 (172.17.0.5:51820)
[  +0.000001] wireguard: wireguard.cali: Sending handshake response to peer 443 (172.17.0.5:51820)
[  +0.000086] wireguard: wireguard.cali: Invalid handshake response from 172.17.0.5:51820
[  +0.000001] wireguard: wireguard.cali: Keypair 136 created for peer 443
[  +0.000025] wireguard: wireguard.cali: Invalid handshake response from 172.17.0.4:51820
[  +5.061132] wireguard: wireguard.cali: Handshake for peer 444 (172.17.0.4:51820) did not complete after 5 seconds, retrying (try 2)
[  +0.000040] wireguard: wireguard.cali: Sending handshake initiation to peer 444 (172.17.0.4:51820)
[  +0.000801] wireguard: wireguard.cali: Receiving handshake initiation from peer 443 (172.17.0.5:51820)
[  +0.000004] wireguard: wireguard.cali: Sending handshake response to peer 443 (172.17.0.5:51820)
[  +0.000709] wireguard: wireguard.cali: Keypair 136 destroyed for peer 443
[  +0.000004] wireguard: wireguard.cali: Keypair 137 created for peer 443
[  +0.000550] wireguard: wireguard.cali: Receiving handshake response from peer 444 (172.17.0.4:51820)
[  +0.000012] wireguard: wireguard.cali: Keypair 138 created for peer 444
```